pandas/tests/arrays/test_integer.py

```diff
@@ -598,6 +604,11 @@ def test_preserve_groupby_dtypes(op):
         "C": integer_array([1, None, 3], dtype='Int64'),
     })
 
+    # op
+    result = getattr(df.C, op)()
+    assert isinstance(result, int)
```

Comment:
Can you then repeat this test for eg mean and assert the above is float (in a case where the mean is a rounded number) ?

Reply:
(that's what I tried to ask with my previous comments on this)